web/beacon-app/src/features/topics/components/RevokeAPIKeyModal.tsx

```diff
@@ -66,24 +66,29 @@ const RevokeAPIKeyModal = ({ onOpen, onClose }: RevokeAPIKeyModalProps) => {
       <>
         <p className="pb-4">
           <Trans>
-            Revoking the API key will result in producers and consumers connected to the topic to{' '}
+            Revoking the API key will result in publishers and subscribers connected to the topic to{' '}
```

Comment:
I saw Edwin's message about updating the copy so included it here since I was making a change to the checkbox.